x-pack/plugins/painless_lab/public/application/components/main_controls.tsx

```diff
@@ -87,12 +79,8 @@ export function MainControls({
     </EuiContextMenuItem>,
   ];
 
-  const classes = classNames('painlessLab__bottomBar', {
```

Comment:
🎉 I love that we don't need this any more! But what about for users who switch back to the old nav?

![image](https://user-images.githubusercontent.com/1238659/80267988-da8ae080-8658-11ea-96e7-7c3177122396.png)

Reply:
Ugh, yes. Good catch, got over eager!